.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "editor.inlineSuggest.showToolbar": "always"
+}
```

Comment:
Also please remove this file. It would be nice to have IDE specific settings relevant to the project (like formatting options) but I think this is more of a personal preference (which might not agree with someone else).

Reply:
I missed to remove it.